thinc/util.py

```diff
@@ -203,12 +203,26 @@ def copy_array(dst: ArrayXd, src: ArrayXd) -> None:  # pragma: no cover
         numpy.copyto(dst, src)  # type: ignore
 
 
-def to_categorical(Y: IntsXd, n_classes: Optional[int] = None) -> FloatsXd:
+def to_categorical(
+    Y: IntsXd,
+    n_classes: Optional[int] = None,
+    *,
+    label_smoothing: float = 0.0,
+) -> FloatsXd:
+    if not 0.0 <= label_smoothing < 0.5:
+        raise ValueError(
+            "label_smoothing should be greater or "
+            "equal to 0.0 and less than 0.5, "
+            "but {} was given".format(label_smoothing)
```

Comment:
we can just use an f-string here:
```suggestion
            f"but {label_smoothing} was given"
```